### PR TITLE
UI Forms

### PR DESCRIPTION
**Pull request by jasonwynn10**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
This PR adds UI forms to MyPlot.

## Optional information
N/A

### CODE REVIEW

File: resources/config.yml [hunk 1]
```diff
@@ -31,6 +31,10 @@ Language: eng
 # This is not recommended unless you have a good text editor!
 Custom Messages: false
 
+# Allows use of MyPlot forms on /plot command
+# This does not add UIs for all commands. Some actions still require commands to use.
+UI Forms: true
+
```
Comment (Aericio) [inline resources/config.yml:37]: > Custom Messages
> UI Forms

spaces reeeeeeeeeeeeeeeeeeeeeeeeeee

Comment (jasonwynn10) [reply resources/config.yml:37]: what are you complaining about?

Comment (Aericio) [reply resources/config.yml:37]: allergic to spaces in config keys!

**Comment by jasonwynn10** [comment]
Closing in favor of using pm-forms virion

**Pull request closed without merging**